WLC/__main__.py

```diff
@@ -1,10 +1,12 @@
 from WLC.code_executor import CodeExecutor
 from WLC.image_processing.camera import Camera
+from WLC.image_processing.image_preprocessor import ImagePreprocessor
```

Comment:
wouldn't it be shorter if we just called it `WLC.image_processing.preprocessor`?

Reply:
Yeah will change.